ibm/service/kubernetes/resource_ibm_container_vpc_cluster.go

```diff
@@ -290,6 +290,13 @@ func ResourceIBMContainerVpcCluster() *schema.Resource {
 				Description:      "Root Key ID for boot volume encryption",
 				RequiredWith:     []string{"kms_instance_id"},
 			},
+			"kms_account_id": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				DiffSuppressFunc: flex.ApplyOnce,
+				Description:      "Account ID of kms instance holder - if not provided, defaults to the account in use",
+				RequiredWith:     []string{"kms_account_id", "crk"},
```

Comment:
If we want to use same account kms still we need to provide this along with crk?
since we added RequiredWith

Reply:
I think we should n't ask for kms_account_id if its from same account